### PR TITLE
"Added Button to login on Settings page"

### LINKED ISSUE #505: Add Option to login on Settings Page
[x] Feature request 
**Actual Behaviour**
There is no link to help the user to log in.

**Expected Behaviour**
There should be link to login.

**Screenshots of the issue**

**Would you like to work on the issue?**

Yes
screenshot from 2018-12-15 21-14-30

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #505 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

-"Added Button to login on Settings page"

screenshot from 2019-01-23 11-03-43

### CODE REVIEW

**Comment by sk9331657** [comment]
@ms10398  @hkedia321  Please review.

**Comment by sk9331657** [comment]
@zamhaq  Please review.

**Comment by sk9331657** [comment]
@hkedia321 @prateekiiest  Please review and merge this.

File: src/js/options.js [hunk 1]
```diff
@@ -26,8 +26,11 @@ window.onload = () => {
         customBackground.style.display = "block";
     }
     else {
-        warning.innerHTML = "Please login for some useful features";
+        warning.innerHTML = "Please login for some useful features <button type='button' id='loginbutton' class='btn btn-success' style='background-color:green !important;color:white !importa'>Login</button>";
```
Comment (hkedia321) [inline src/js/options.js]: @sk9331657 is `!important` needed here?

Comment (sk9331657) [reply src/js/options.js]: @hkedia321  To override already present css.

Comment (hkedia321) [reply src/js/options.js]: @sk9331657 It is not needed here. The inline CSS already override the already present CSS in the file. You can remove it.

Comment (sk9331657) [reply src/js/options.js]: Done @hkedia321  . Ready to be merged.

**Comment by sk9331657** [comment]
@anubhavp28 @mariobehling  Please review.

**Comment by sk9331657** [comment]
@hkedia321  @fragm3  Added Screenshot.

**Pull request merged**